Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1974,20 +1973,18 @@ def get_remote_data_command(client: Client, args: Dict[str, Any], params: Dict)
             required=False
         )
         demisto.debug(f'entry_time is {entry_time}')
-
-        if last_update > entry_time:  # type: ignore
-            continue
-
-        comments_context = {'comments_and_work_notes': note.get('value')}
-        entries.append({
-            'Type': note.get('type'),
-            'Category': note.get('category'),
-            'Contents': note.get('value'),
-            'ContentsFormat': note.get('format'),
-            'Tags': note.get('tags'),
-            'Note': True,
-            'EntryContext': comments_context
-        })
+        if 'Mirrored from Cortex XSOAR' not in note.get('value'):
+            if entry_time > last_update:
```

Comment:
same as in attachments, let not query for all comments, but only ones that were created after the last update time and not iterate all of them